workitems/tokenlist/l2-token-list-v1.0-psd01.md

```diff
@@ -0,0 +1,723 @@
+                            "type": "string",
+                            "examples": ["https://polygonscan.com/address/0xa9c28ce2141b56c474f1dc504bee9b01eb1bd7d1a507580d5519d4437a97de1b"]
+                        },
+                        "tokenIssuerName": {
+                            "$comment": "{\"term\": \"tokenIssuerName\", \"@id\": \"https://schema.org/name\"}",
+                            "title": "tokenIssuerName",
+                            "description": "The name oof the token issuer.",
+                            "type": "string"
+                            "examples": ["Matic"]
+                        },
+                        "tokenId": {
+                            "$comment": "{\"term\": \"tokenId\", \"@id\": \"https://schema.org/identifier\"}",
+                            "title": "tokenId",
+                            "description": "A resolvable URI to a CanonicalTokenList entry of the token following the RFC 3986 standard.",
+                            "type": "string",
+                            "example": ["https://polygonscan.com/address/0x0000000000000000000000000000000000001010"]
+                        },
+                        "tokenType": {
+                            "$comment": "{\"term\": \"tokenType\", \"@id\": \https://schema.org/StructuredValue\"}",
+                            "title": "tokenType",
+                            "description": "Describes the type of token.",
+                            "type": "array"
+                            "examples"[["fungible","transferable"]]
+                        },
+                        "tokenDesc": {
+                            "$comment": "{\"term\": \"tokenDesc\", \"@id\": \"https://schema.org/description\"}",
+                            "title": "tokenDesc",
+                            "description": "Brief description of the token and its functionality.",
+                            "type": "string",
+                            "examples": ["Protocol Token for the Matic Network"]
+                        },
+                        "standard": {
+                            "$comment": "{\"term\": \"standard\", \"@id\": \"https://schema.org/citation\"}",
+                            "title": "standard",
+                            "description": "Describes which standard the token adheres to.",
+                            "type": "string",
+                            "examples": ["ERC20"]
+                        },
+                        "address": {
+                            "$comment": "{\"term\": \"address\", \"@id\": \"https://schema.org/identifier\"}",
+                            "title": "address",
+                            "description": "Address of the token smart contract.",
+                            "type": "string",
+                            "examples": ["0x0000000000000000000000000000000000001010"]
+                        },
+                        "addressType": {
```

Comment:
Also don't quite understand the purpose of this

Reply:
Discussed during 9/7/22 WG meeting and resolved. Makes sense for non-EVM L2s